### PR TITLE
SIGs: Remove some SIGs

### PR DESCRIPTION
**Pull request by fabiand**
**What this PR does / why we need it**:
Remove some unusued sigs, focus on the essentials

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
sig-virtctl not in use: https://github.com/search?q=org%3Akubevirt%20sig-virtctl&type=code
sig virt not in use: https://github.com/search?q=org%3Akubevirt%20sig-virt&type=code
sig userinterface not in use: https://github.com/search?q=org%3Akubevirt%20sig-userinterface&type=code

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [x] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [x] PR: The PR description is expressive enough and will help future contributors
- [x] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [x] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [x] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [x] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [x] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [x] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by fabiand** [comment]
@aburdenthehand @dhiller please review

File: sigs.yaml [hunk 1]
```diff
@@ -53,18 +53,6 @@ sigs:
           owners:
             - https://raw.githubusercontent.com/kubevirt/kubevirtci/main/OWNERS
             - https://raw.githubusercontent.com/kubevirt/project-infra/main/OWNERS
-    - dir: sig-virtctl
```
Comment (dhiller) [inline sigs.yaml:56]: Reminder: the SIGs defined in this file are also used as base for the repository groups inside `devstats` ([example](https://kubevirt.devstats.cncf.io/d/1/activity-repository-groups?orgId=1), take a look at the Repository Group selector, containing `SIG virtctl`

So, if we are removing this sig, we should move the OWNERS pointer to maybe the `sig-compute`.

Comment (fabiand) [reply sigs.yaml:56]: Ack, thanks!

**Comment by fabiand** [comment]
Comment addressed, thanks

**Comment by aburdenthehand** [comment]
/lgtm
I can't think of a compelling reason to keep these. Thanks @fabiand

**Comment by aburdenthehand** [comment]
/hold
@dhiller @fabiand Do we need to worry about this failing devstats test?

**Comment by fabiand** [comment]
Good catch, I gave it a go, lets see if it passes.

**Comment by fabiand** [comment]
@aburdenthehand works now!

/unhold

**Pull request merged**